rmgpy/rmg/main.py

```diff
@@ -1210,6 +1196,62 @@ def readMeaningfulLineJava(self, f):
         return line
     
 ################################################################################
+def processToSpeciesNetworks(obj,reactionSystem,coreSpecies):
+    """
+    breaks down the objects returned by simulate into Species and PDepNetwork
+    components
+    """
+    if isinstance(obj, PDepNetwork):
+        out = [processPdepNetworks(obj,reactionSystem)]
+        return out
+    elif isinstance(obj, Species):
+        return [obj]
+    elif isinstance(obj,Reaction):
+        return list(processReactionsToSpecies(obj,coreSpecies))
+    elif isinstance(obj,list): #list of species
+        rspcs = processReactionsToSpecies([k for k in obj if isinstance(k,Reaction)],coreSpecies)
+        spcs = list({k for k in obj if isinstance(k,Species)} | rspcs)
+        nworks = list(set(processPdepNetworks([k for k in obj if isinstance(k,PDepNetwork)],reactionSystem)))
+        return spcs+nworks
+    else:
+        assert False, "improper call, obj input was incorrect"
+            
+def processPdepNetworks(obj,reactionSystem):
+    """
+    properly processes PDepNetwork objects and lists of PDepNetwork objects returned from simulate
+    """
+    if isinstance(obj, PDepNetwork):
+        # Determine which species in that network has the highest leak rate
+        # We do this here because we need a temperature and pressure
+        # Store the maximum leak species along with the associated network
+        ob = (obj, obj.getMaximumLeakSpecies(reactionSystem.T.value_si, reactionSystem.P.value_si))
+        return ob
+    elif isinstance(obj,list):
+        return [(ob, ob.getMaximumLeakSpecies(reactionSystem.T.value_si, reactionSystem.P.value_si)) for ob in obj]
+    else:
+        assert False, "improper call, obj input was incorrect"
+        
+def processReactionsToSpecies(obj,coreSpecies):
+    """
+    properly processes Reaction objects and lists of Reaction objects returned from simulate
+    """
+    filterFcn = lambda x: not ((x in coreSpecies)) #remove species already in core
+    if isinstance(obj,Reaction):
+        potentialSpcs = obj.reactants+obj.products
+        potentialSpcs = filter(filterFcn,potentialSpcs)
+    elif isinstance(obj,list) or isinstance(obj,set):
+        potentialSpcs = set()
+        for ob in obj:
+<<<<<<< HEAD
```

Comment:
Can wherever you fix this merge conflict be squashed to this commit when you next rebase?